### PR TITLE
Feature/134 latest amends to scraped vacancies

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

**Comment by despo** [comment]
[Latest amends to scraped vacancies](https://trello.com/c/XNYq5xUX/134-latest-amends-to-scraped-vacancies)

File: lib/tasks/vacancies_to_update.yaml [hunk 1]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <ul><li> have exceptional planning and time management skills</li>
+      <li>be able to inspire and challenge both students and colleagues</li>
+      <li>have the ability to communicate effectively with a wide range of stakeholders including students, parents, the media and governors</li>
+      <li>have evidence of outstanding results and student outcomes</li>
+      <li> and be committed to working in an inclusive environment in support of the values, vision, purpose and direction of Durham Sixth Form Centre</li></ul>"
+      starts_on: '01/09/2018'
+      ends_on: '31/08/2019'
+    -
+      slug: 'head-of-history'
+      job_title: 'Head of History'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 2]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p> Any appointment is subject to an enhanced D.B.S. check. Please note that the Childcare Disqualifications Regulations 2009 apply to this position and therefore you are required to complete the “disqualification by association” declaration form. Should you be successful in your application for this post, your appointment can not be progressed without this declaration.</p>
+      <p>The Governing Body is an Equal Opportunity Employer.</p>
+      <p>Applicants with disabilities will be invited for interview if the essential job criteria are met.</p>
+      <p> This post is not open to job share.<br/> DIOCESE OF HEXHAM AND NEWCASTLE<br> DURHAM MARTYRS MULTI ACADEMY TRUST</p>"
+      subject: 'History'
+      benefits: 'TLR 2c (£6515)'
+      experience: 'The post provides an exciting opportunity for a forward-thinking and well-motivated classroom practitioner with proven success at GCSE and A Level. The ability to teach A level Government and Politics would be an advantage'
+    -
+      slug: 'teacher-of-pe-girls'
+      job_title: 'Teacher of PE (Girls)'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 3]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>GSHS is committed to safeguarding and promoting the welfare of children and young people and expects all staff and volunteers to share this commitment.</p> "
+      subject: 'Physical education'
+      experience: "We would love to hear from you if you have:
+      <ul><li>passion and energy to inspire and motivate students and staff</li>
+      <li>a track record of success in raising standards</li>
+      <li>exceptional interpersonal skills </li>
+      <li>a strong commitment to teamwork</li></ul>"
+    -
+      slug: 'teacher-of-history-maternity-cover'
+      job_description: "
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 4]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+Any appointment is subject to an enhanced D.B.S. check. Please note that the Childcare Disqualifications Regulations 2009 apply to this position and therefore you are required to complete the “disqualification by association” declaration form. Should you be successful in your application for this post, your appointment can not be progressed without this declaration.</p>
+      <p>The Governing Body is an Equal Opportunity Employer.</p>
+      <p>Applicants with disabilities will be invited for interview if the essential job criteria are met.</p>
+      <p>This post is not open to job share.</p>"
+      experience: 'Applicants should be a well-qualified teacher of History able to teach Key Stage 3, GCSE and the ability to teach A level would be an advantage'
+      qualifications: 'Qualified teacher status (or on course to obtain one) and a good honours degree'
+      max_pay_scale: 'Upper Pay Scale 3'
+    -
+      slug: 'teacher-of-science'
+      job_title: 'Teacher of Science'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 5]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>Framwellgate School Durham is committed to the safeguarding of children and young people. An enhanced disclosure from the DBS will be required for this post, together with completion of a pre-employment health questionnaire.</p>
+      <p>The Excel Academy Partnership at Framwellgate School Durham<br> Newton Drive<br> Durham<br> DH1 5BQ<br> Tel: (0191) 3866628<br> Email: Thompson.f@framdurham.com</p>"
+      subject: 'Art'
+      benefits: 'We can offer you an environment where student behaviour is excellent, prior attainment levels are amongst the highest in the North-East and our students and staff are a pleasure to work with'
+      experience: 'The successful candidate will possess energy, vision and enthusiasm and believe that every young person deserves the very best education. A commitment to raising achievement, and a willingness to learn, train and develop as a teacher are all essential.'
+      working_pattern: 'part_time'
+    -
+      slug: 'teacher-of-geography-0-8-1-0'
+      job_title: 'Teacher of Geography (0.8 to 1.0)'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 6]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <li>able to motivate students</li>
+      <li>committed to continuing professional development</li></ul>
+      <p>Potential candidates are welcome to contact Matt Snape, Headteacher, for an informal briefing on 0191 2962771.</p>
+      <p>Closing date: 9am/first post on Friday 18 May 2018<br> Interviews:     Thursday 24 May 2018</p>
+      <p>Application packs can be downloaded from our website: www.mardenhigh.net</p>
+      <p>Marden High School is committed to safeguarding and promoting the welfare of children and young people. The successful applicant will be required to undertake an enhanced DBS disclosure and a range of other recruitment checks.</p>"
+      subject: "Science"
+    -
+      slug: 'teacher-of-english'
+      job_title: 'Teacher of English'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 7]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>Any appointment is subject to an enhanced D.B.S. check. Please note that the Childcare Disqualifications Regulations 2009 apply to this position and therefore you are required to complete the “disqualification by association” declaration form. Should you be successful in your application for this post, your appointment can not be progressed without this declaration.</p>
+      <p>The Governing Body is an Equal Opportunity Employer.</p>
+      <p>Applicants with disabilities will be invited for interview if the essential job criteria are met.</p>
+      <p>This post is not open to job share.<br>
+      DIOCESE OF HEXHAM AND NEWCASTLE<br>
+      DURHAM MARTYRS MULTI ACADEMY TRUST</p>"
+      subject: "English"
+    -
+      slug: 'curriculum-leader-of-mathematics'
+      job_title: "Curriculum Leader of Mathematics"
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 8]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>If you would like any further information, or to discuss this role in more detail, please contact Daniel Jamieson, Headteacher.</p>
+      <p>In addition to the application form please provide a covering letter, of no more than two sides of A4, in which you outline your suitability for the post as well as your philosophy on teaching.  Completed applications should be returned to Mrs L Heide, Burnside Business and Enterprise College, St. Peter’s Road, Wallsend, NE28 7LQ or electronically to l.heide@burnsidecollege.org.uk</p>
+      <p>Telephone: 0191 2598500 </p>
+      <p>Burnside is committed to safeguarding and all offers of employment are subject to satisfactory enhanced DBS clearance.</p>
+      <p>Closing date:  9am Monday 14th May 2018</p>
+      <p>Interview date: Thursday 17th May 2018</p>"
+      experience: "The successful candidate will have a strong track record in leadership, and a passion for teaching and learning. In addition, you will have experience of delivering consistently excellent mathematics teaching across key stages"
+    -
+      slug: 'teacher-of-history-maternity-cover'
+      job_title: 'Teacher of History (part time maternity cover)'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: 1) Delete duplicate copy of this listing
2) Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 9]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>Applicants should complete the standard application form and submit a short letter that details the position they are interested in applying for (for example: part-time or full-time); and how their skills would meet the needs of the post.</p>
+      <p>The closing date for applications is Wednesday 2nd May 2018 at 12.30pm. The provisional interview date is Thursday 17th May 2018.</p>
+      <p><em>‘This school is committed to safeguarding and promoting the welfare of children and young people and expects all staff and volunteers to share this commitment’</em></p>"
+      experience: "The successful candidate will:
+      <ul><li>have exceptional planning and time management skills</li>
+      <li>be able to inspire and challenge both students and colleagues</li>
+      <li>have the ability to communicate effectively with a wide range of stakeholders including students, parents, the media and governors</li>
+      <li>have evidence of outstanding results and student outcomes</li>
+      <li>be committed to working in an inclusive environment in support of the values, vision, purpose and direction of Durham Sixth Form Centre</li></ul>"
+      starts_on: '01/09/2018'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: delete starts_on entry unless we can just specify "September 2018"

File: lib/tasks/vacancies_to_update.yaml [hunk 10]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>Closing date for the receipt of applications is 12 noon on Friday 11th May 2018.</p>
+      <p>The school is committed to safeguarding and promoting the welfare of children and young people and expects all staff and volunteers to share this commitment.</p>
+      <p>Any appointment is subject to an enhanced D.B.S. check. Please note that the Childcare Disqualifications Regulations 2009 apply to this position and therefore you are required to complete the “disqualification by association” declaration form. Should you be successful in your application for this post, your appointment can not be progressed without this declaration.</p>
+      <p>The Governing Body is an Equal Opportunity Employer.</p>
+      <p>Applicants with disabilities will be invited for interview if the essential job criteria are met.</p>
+      <p>This post is not open to job share.</p>
+      <p>DIOCESE OF HEXHAM AND NEWCASTLE<br> DURHAM MARTYRS MULTI ACADEMY TRUST"
+    -
+      slug: 'teacher-of-health-and-social-care'
+      job_title: 'Teacher of Health and Social Care'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 11]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      experience: "The successful candidate will:
+      <ul><li>have exceptional planning and time management skills</li>
+      <li>be able to inspire and challenge both students and colleagues</li>
+      <li>have the ability to communicate effectively with a wide range of stakeholders including students, parents, the media and governors</li>
+      <li>have evidence of outstanding results and student outcomes</li>
+      <li>be committed to working in an inclusive environment in support of the values, vision, purpose and direction of Durham Sixth Form Centre</li></ul>"
+      starts_on: '01/09/2018'
+    -
+      slug: "teacher-of-ict-computing"
+      job_title: "Teacher of ICT"
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 12]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>An Ofsted “Good School”, our most recent Ofsted report (January 2018) judged that “Pupils conduct themselves very well in lessons and around the school. They are confident, polite and courteous to one another and to staff. Pupils behave well and show positive attitudes to learning“ and that  “Leaders ensure that learning in subjects is effectively enhanced through the wide range of sporting and after-school activities on offer.“</p>
+      <p>The PE Department has a strong reputation both within the school and within the North East where we have a history of success in extra-curricular sport.</p>
+      <p>The Governing Body are committed to safeguarding and promoting the welfare of children and young people and vulnerable adults. The successful candidate will be subject to an enhanced disclosure through the Disclosure and Barring Service and additional recruitment checks.</p>
+      <p>NQTs are welcome to apply.</p>
+      <p>NB – Applications close Friday 11th May 2018.  Interviews will take place on Thursday 17th May.</p>
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Delete 'Applications close Friday 11th May 2018.' (indicated in box, and no specific time given so no added value)

File: lib/tasks/vacancies_to_update.yaml [hunk 13]
```diff
@@ -212,3 +212,477 @@ We can offer:
...
+      <p>All post information can be downloaded either from this advert or from http://www.burnsidecollege.org.uk/vacancies/assistant-curriculum-leader-of-english-mps-ups-tlr-2c</p>
+      <p>In addition to the application form please provide a covering letter, of no more than two sides of A4, in which you outline your suitability for the post as well as your philosophy on teaching.  Completed applications should be returned to Mrs L Heide, Burnside Business and Enterprise College, St. Peter’s Road, Wallsend, NE28 7LQ or electronically to <strong>l.heide@burnsidecollege.org.uk</strong> If you would like a tour of the school, or to discuss any aspect of the role, please don’t hesitate to contact Mrs E. Davis at e.davis@burnsidecollege.org.uk<br>
+Telephone: 0191 2598500</p>
+      <p>Burnside is committed to safeguarding and all offers of employment are subject to satisfactory enhanced DBS clearance.</p>
+      <p>Closing date:  9am Friday 4th May 2018<br>
+      Interview date:  Thursday 10th May 2018</p>"
+      starts_on: '01/09/2018'
+    -
+      slug: 'teacher-of-maths-maternity-cover'
+      job_title: 'Teacher of Maths (maternity cover)'
```
Comment (mnottingham-dfe) [inline lib/tasks/vacancies_to_update.yaml]: Pay scale range should be Main pay range 1 to Upper pay range 3

File: lib/tasks/vacancies_to_update.yaml [hunk 14]
```diff
@@ -212,3 +213,485 @@ We can offer:
...
+      <li>knowledge of effective assessment practice to ensure progress is tracked and informs further development for each individual</li>
+      <li>clear commitment towards raising standards and expectations to enable all learners to reach their potential</li>
+      <li>excellent organisation skills with a commitment to professional development</li>
+      <li>a demonstrable record of outstanding classroom practice – commitment, dedication and enthusiasm in sharing our vision for the development of our community</li>
+      <li>a commitment to child protection procedures and policy and an understanding of safeguarding</li></ul>'
+      qualifications: 'Qualified Teacher Status'
+      starts_on: "01/09/2018"
+      ends_on: "31/08/2019"
+    -
+      slug: 'english-literacy-amp-enrichment-co-ordinator'
```
Comment (tahb) [inline lib/tasks/vacancies_to_update.yaml]: When there are a subset of fields like this does it mean that the only update to do for this vacancy is the job_description?

File: lib/tasks/vacancies_to_update.yaml [hunk 15]
```diff
@@ -212,3 +213,485 @@ We can offer:
...
+      <p>Exciting opportunities have arisen for confident and knowledgeable individuals to join our professional and highly skilled team. We are keen to employ enthusiastic Support Assistants for the following roles:
+      <ul><li> dedicated support for science department (must have studied A Level science)</li>
+      <li>general classroom support duties</li></ul></p>
+      Both roles are part time (32.5 hours over 5 days per week) and involve classroom support, individual and small group work in close collaboration with other school staff and supporting professionals. In return we offer the opportunity to work in an innovative and supportive environment.
+      <p> Application forms and further details are available on the council website<br>
+      www.newcastle.gov.uk and returnable by email to admin@benfield.newcastle.sch.uk<br>
+      or by post to Mrs L Reeson, Benfield School, Benfield Road, Newcastle NE6 4NU.</p>
+      <p>Please note CVs not accepted.<br>
+      Closing Date:            9.00am Tuesday 8th May 2018<br>
```
Comment (tahb) [inline lib/tasks/vacancies_to_update.yaml]: Is this a field we are happy to put into the right place in the `ends_on` field?

File: lib/tasks/vacancies_to_update.yaml [hunk 16]
```diff
@@ -212,3 +213,485 @@ We can offer:
...
+      <p><em>“Benfield is a good school that has improved considerably since its previous inspection“<br>
+      Teachers have good subject knowledge and use this to support learning well. They are enthusiastic and use some innovative techniques that enthuse students”<br>
+      <strong>(Ofsted May 2015)</strong></em></p>
+      <p>Applications are welcomed from both newly-qualified and experienced teachers.</p>
+      <p>Further details and an application pack, can be found on the council website at www.newcastle.gov.uk or email: lilian.reeson@benfield.newcastle.sch.uk</p>
+      <p>Completed applications should be returned to Mrs L Reeson, PA to Headteacher, either by email or post.<br>
+      Email: lilian.reeson@benfield.newcastle.sch.uk <br>
+      Mrs L Reeson, Benfield School, Benfield Road, Newcastle, NE6 4NU</p>
+      <p>Please note CVs are not accepted.<br>
+      <strong>Closing date:                           9.00am Tuesday 8th May 2018<br>
```
Comment (tahb) [inline lib/tasks/vacancies_to_update.yaml]: Another closing date that could go into `ends_on`.

File: lib/tasks/vacancies_to_update.yaml [hunk 17]
```diff
@@ -212,3 +213,485 @@ We can offer:
...
+      education: "The successful candidate will be someone who has:
+      <ul><li>a passion for providing the best possible education for all students</li>
+      <li>a commitment to maximising achievement through excellence in teaching and learning</li>
+      <li>a strong commitment to the Catholic ethos of the Trust and the promotion of Christian values</li>
+      <li>a commitment to continual improvement</li>
+      <li>the ability to inspire students and promote a sense of enjoyment of learning</li>
+      <li>an eagerness to be involved with extra-curricular activities in order to enhance the lives of the whole community</li></ul>"
+    -
+      slug: 'teacher-of-physics'
+      job_descritpion: "
```
Comment (tahb) [inline lib/tasks/vacancies_to_update.yaml]: This is the entry that my go of the rake task crashed on.

File: lib/tasks/vacancies_to_update.yaml (hunk 16 above)
Comment (despo) [reply lib/tasks/vacancies_to_update.yaml]: @tahb closing date is the application closing date, whilst ends_on is the job end date.

File: lib/tasks/vacancies_to_update.yaml (hunk 14 above)
Comment (despo) [reply lib/tasks/vacancies_to_update.yaml]: yes, this job specifically only needs the title updated.

File: lib/tasks/vacancies_to_update.yaml (hunk 15 above)
Comment (despo) [reply lib/tasks/vacancies_to_update.yaml]: No. We don't capture the application closing date.

File: lib/tasks/vacancies_to_update.yaml (hunk 17 above)
Comment (despo) [reply lib/tasks/vacancies_to_update.yaml]: 🙄

**Comment by despo** [comment]
@tahb updated

**Pull request merged**